### ISSUE TITLE
HTTPS Not Sure and unsafe issue with URL

### ISSUE THREAD

**Issue by linkzeeshan** [jitsi/docker-jitsi-meet#1806]
Can you guide me how can I resolve https not secure issue. SSL is registered on my domain. but still not secure issue is coming.

I purchased VM on Azure and domain purchase to another platform.

**Comment by saghul** [comment]
What public URL are you using?

**Comment by linkzeeshan** [comment]
Currently I am using VM IP

**Comment by saghul** [comment]
Test your domain here: https://whatsmychaincert.com/

**Comment by linkzeeshan** [comment]
Please check screenshot.

**meet2.medchois.com is self-signed. It doesn't have a chain certificate and will never be trusted by clients.**

![image](https://github.com/jitsi/docker-jitsi-meet/assets/66298499/1857081b-bd70-4651-874e-67ebcad6062e)

**Comment by damencho** [comment]
Have you enable the 3 let's encrypt settings?
https://github.com/jitsi/docker-jitsi-meet/blob/731d6d13b88b9c74877312f2f0d0250e9894de18/env.example#L60

**Comment by linkzeeshan** [comment]
[damencho](https://github.com/damencho) I checked this  #ENABLE_LETSENCRYPT=1  It was commented.

I uncommented now page is not loading please check below screenshot

 #ENABLE_LETSENCRYPT=1

**Comment by saghul** [comment]
Check the web container logs.

**Comment by damencho** [comment]
> [damencho](https://github.com/damencho) I checked this #ENABLE_LETSENCRYPT=1 It was commented.

You also need the following two, the domain and email.

**Comment by linkzeeshan** [comment]
[saghul](https://github.com/saghul) [damencho](https://github.com/damencho) All done but still same issue is coming

let me try to explain once 
1- site is working but once I set below configuration

PUBLIC_URL = https://IP-address
ENABLE_LETSENCRYPT, domain and email. all option are commented 

2- site not working once I enable below following options  
ENABLE_LETSENCRYPT = 1
LETSENCRYPT_DOMAIN=domain name
LETSENCRYPT_EMAIL= email

**Comment by damencho** [comment]
> PUBLIC_URL = https://IP-address
This should be https://domain name. 


You should have a valid DNS for that domain name in order Let's Encrypt to reach your server and allow you to have valid certificates, it also needs port 80 to be accessible on your deployment. 
As Saul said you can see more information on what went wrong with the certificates in the logs of the container.

**Comment by linkzeeshan** [comment]
Thanks. 
I assigned PUBLIC_URL Value is  my domain name https://domain/

Please check below log SC.

![image](https://github.com/jitsi/docker-jitsi-meet/assets/66298499/ec39dfc3-c5fc-4c79-900f-113ec4e8eefc)

**Comment by linkzeeshan** [comment]
![image](https://github.com/jitsi/docker-jitsi-meet/assets/66298499/d123563f-753b-443f-ad5a-03a5f2ff98b5)

**Comment by saghul** [comment]
Is the domain DNS pointing at the correct IP address and are ports 80 and 443 forwarded to the docker host?

**Comment by saghul** [comment]
Please share the full .env file.

**Comment by linkzeeshan** [comment]
Please find attached screens shot
![image](https://github.com/jitsi/docker-jitsi-meet/assets/66298499/69f47ce4-276d-4c6f-a60c-449df202b9ad)

**Comment by saghul** [comment]
Is your public URL correct in your file?

On a previous screenshot I noticed you are running Docker as root. Don't, use a regular user please.

Even if LE fails, are you able to access the site over plain http?

**Comment by linkzeeshan** [comment]
No! It's not working with plain http as well.

PUBLIC_URL = htpps://meet2.medchois.com

**Comment by damencho** [comment]
Are you sure the ports are forwarded correctly and the traffic from the public IP reaches the running docker?

**Comment by linkzeeshan** [comment]
Yes! I have already opened the inbound connection, and the site is functioning properly once I set the value of PUBLIC_URL as the VM IP and commented out the other keys listed below:

ENABLE_LETSENCRYPT = 1
LETSENCRYPT_DOMAIN = domain name
LETSENCRYPT_EMAIL = email

I also completed the DNS configuration for the domain name.

**Comment by saghul** [comment]
Hum, so it only fails when you enable let's encrypt?

Try this: put the domain in there and leave let's encrypt disabled. Does the site work, albeit with the cert error?

**Comment by linkzeeshan** [comment]
Yes [saghul](https://github.com/saghul), the application  functioning correctly with the configuration set as follows:

PUBLIC_URL = https://vm/ Ip
ENABLE_LETSENCRYPT = disabled

However, I encountered an issue when enabling LETSENCRYPT and changing the PUBLIC_URL to https://domain/. The application is not working in this configuration.

**Comment by saghul** [comment]
That's not what I asked. Try this:

PUBLIC_URL = https://domain
ENABLE_LETSENCRYPT = disabled

I suspect your domain is not pointing to the right IP.

**Comment by linkzeeshan** [comment]
Yes!

**Comment by damencho** [comment]
So then the ports are not forwarded from the public IP to the vm.
